.editorconfig

```diff
@@ -7,6 +7,10 @@ indent_style = tab
 intent_style = space
 indent_size = 2
 
+[*.cs]
+intent_style = tab
+indent_size = 4
```

Comment:
You've got an mistake here, `intent` vs `indent`. Looks like it is also wrong under csproj which I assumed you copy-pasted.